doc/guides/getting_started/src/unofficial/fedora.md

```diff
@@ -0,0 +1,19 @@
+# Unofficial Setup Guide for Fedora
+
+This is an unofficial guide detailing how to set up a Fedora system for OpenTitan development.
+
+## Step 2: Install dependencies using the package manager
+
+Fedora uses a different package manager than the officially supported Ubuntu development environment.
+You can install equivalent packages to the `apt-requirements.txt` file with the following command:
```

Comment:
Hmm, it's a bit of a pity to have the list of files in two places, because this guide will bit-rot as soon as our dependencies change. I don't suppose there's a nice way to get the Redhat names from the Debian ones?

Reply:
I don't know of any automated way to do this.  I usually have to consult a search engine package-by-package.